### ISSUE TITLE
Error while starting the local preview.

### ISSUE THREAD

**Issue by meet7-sagar23** [AgoraIO-Extensions/Agora-Flutter-SDK#828]
https://github.com/AgoraIO-Extensions/Agora-Flutter-SDK/blob/1bdb923668e1be27541cb5c331e2daa30d4079ec/example/lib/examples/basic/join_channel_video/join_channel_video.dart#L100

Please run the below code snippet to reproduce.

```dart
try {
  await _engine.startPreview();
} catch (e) {
  print(e);
}
```

Here is the output: -
> e
> AgoraRtcException (AgoraRtcException(-8, null))
> code: -8
> message: null
> hashCode: 674072757
> runtimeType: Type (AgoraRtcException)

**Comment by amityamsi** [comment]
This error occurs because of 
The specified view is invalid. The video call function requires a specified view. 

read agora error and code docs here
https://docs.agora.io/en/3.x/video-calling/reference/error-codes?platform=flutter

**Comment by littleGnAl** [comment]
I think more info is needed

**Comment by xmbebebol** [comment]
Any updates? Also getting the same error.

**Comment by xmbebebol** [comment]
Error code details: "8 | The specified view is invalid. The video call function requires a specified view." What does this mean?

**Comment by VijaiCPrasad** [comment]
This seem to happen randomly when trying to accept the call from background. Using agora_rtc v6.1.0

**Comment by maerlynflagg** [comment]
I also noticed the bug in version 6.1.0 and 6.1.1. It occurs, for example, when I use "switchCamera" or when I set the second camera to "startPreview" as the video source. Both in the emulator and on real Android devices, the app always crashes with error 8 as soon as the front camera is referred to. If I use the selfie camera, the error does not appear.

If I start with the selfie camera and then change the camera with a button click, then that works. Only starting with the right camera doesn't work.
